python/simulation/fake_raft.py

```diff
@@ -0,0 +1,509 @@
+"""
```

Comment:
Ok, done.   I'm not sure how far you want to take this.   I did everything pylint suggested, included stuff like changing the variable names and reducing the number of arguments / local variables in functions.